src/components/sidenav/sidenav.js

```diff
@@ -365,6 +367,8 @@ function SidenavDirective($mdMedia, $mdUtil, $mdConstant, $mdTheming, $animate,
       ]).then(function() {
         // Perform focus when animations are ALL done...
         if (scope.isOpen) {
+          // Notifies child components that the sidenav was opened.
+          ngWindow.triggerHandler('resize');
```

Comment:
It's not really a resize event, it just fires all of the resize handlers. An alternative to this can be to fire off a `$md-resize` event which is only for the virtual repeaters. I decided to go with this so other components could take advantage of it.